Lint failures do not describe which file caused the issue

The new linting workflow is trapping warnings and errors, but doesn't say which file or where in the file the violation is.  This makes debugging a terrible experience.

e.g. I have no idea where to even start trying to find these violations.

![image](https://user-images.githubusercontent.com/5819622/169740613-852eff80-7383-444d-96f0-8572603eb617.png)

**Expected Behaviour**

Linting error output should list which file and location within that file caused the failures.

If I run the linter using a local docker container, it does output location information

```
tfe/provider_test.go:380:11: sprintfQuotedString: use %q instead of "%s" for quoted strings (gocritic)
                Config: fmt.Sprintf("resource \"tfe_organization\" \"foobar\" {\n name = \"%s\"\nemail = \"admin@company.com\"\n}", organizationName),
```

--- a/.github/workflows/lint.yml
+++ b/.github/workflows/lint.yml
@@ -21,3 +21,4 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           version: v1.45
+          args: "--out-${NO_FUTURE}format colored-line-number"

